CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ This section is for maintaining a changelog for all breaking changes for the cli
 - Document HTTP/2 support ([#330](https://github.com/opensearch-project/opensearch-java/pull/330))
 - Add support for phase_took & search_pipeline request params ([#1036](https://github.com/opensearch-project/opensearch-java/pull/1036))
 - Add an interface PlainJsonSerializable inherit from JsonpSerializable with a default method streamlining serialization ([#1064](https://github.com/opensearch-project/opensearch-java/pull/1064))
+- Add a new file json.md in guides and SerializationBasics in samples/json ([#1083](https://github.com/opensearch-project/opensearch-java/pull/1083))
```

Comment:
Nit: I would collapse the two changelog items and just say that we added request serialization to JSON.